go.mod

```diff
@@ -1,8 +1,8 @@
 module github.com/pulumi/pulumi-docker-build
 
-go 1.22.0
+go 1.23.1
 
-toolchain go1.23.4
+toolchain go1.24.0
```

Comment:
Can we use 1.23 here? Does this imply that I would need to install 1.24 on my Mac?

Reply:
I removed the `toolchain` directive in a follow-up commit. This will default to the default toolchain installed instead, for simplicity.

> Does this imply that I would need to install 1.24 on my Mac?

No. Go will download the latest toolchain at runtime and use it. [Ref](https://go.dev/doc/toolchain).